README.md

```diff
@@ -208,84 +115,44 @@ Example:
 
 > Mention all of the collaborators of this project.
 
```

Comment:
- [ ] Kindly remove this section description to make your readme file descriptive and clean :blush:

Reply:
Thanks.